### DISCUSSION TITLE
How to render "markdown style" quote

### DISCUSSION THREAD

**Discussion by dignifiedquire** [emilk/egui#1423] [General]
I have some text using `ui.label(RichText(...))` that I am wrapping usig `ui.horizontal_wrapped`. Now I would like to prefix this with an ident and a vertical line, like this (just a random example from the internet). 

![grafik](https://user-images.githubusercontent.com/790842/160253058-105313d3-f7e1-4fd4-a7ff-8b522b51f562.png)

The issue I run into, is that using `ui.separator()` I only get a vertical line for single line, but not for multiple lines. Any pointers on how to do this?

**Comment by dignifiedquire** [comment]
Example from the playground of what I do not want to happen: 

![Screen Shot 2022-03-26 at 19 56 12](https://user-images.githubusercontent.com/790842/160253363-934cdb9a-c6e5-4b0c-918d-a40780387383.png)

**Reply by emilk** [reply]
A hot tip is to look at the source code for the playground :)
https://github.com/emilk/egui/blob/master/egui_demo_lib/src/easy_mark/easy_mark_viewer.rs#L57-L64
You can see that I simply paint the line with `ui.painter().line_segment(…)`

**Reply by dignifiedquire** [reply]
thanks, I did, the challenge I have is to calculate the correct height, as that code assumes a fixed single row.

**Reply by emilk** [reply]
Lay out the text first (e.g. with https://docs.rs/egui/latest/egui/struct.Painter.html#method.layout), measure it, and then paint?
